pkg/skaffold/tag/tag.go

```diff
@@ -29,18 +30,21 @@ type ImageTags map[string]string
 // Tagger is an interface for tag strategies to be implemented against
 type Tagger interface {
 	// GenerateTag generates a tag for an artifact.
-	GenerateTag(workingDir, imageName string) (string, error)
+	GenerateTag(image latest.Artifact) (string, error)
 }
 
 const DeprecatedImageName = "_DEPRECATED_IMAGE_NAME_"
 
 // GenerateFullyQualifiedImageName resolves the fully qualified image name for an artifact.
 // The workingDir is the root directory of the artifact with respect to the Skaffold root,
 // and imageName is the base name of the image.
-func GenerateFullyQualifiedImageName(t Tagger, workingDir, imageName string) (string, error) {
+func GenerateFullyQualifiedImageName(t Tagger, image latest.Artifact) (string, error) {
 	// Supporting the use of the deprecated {{.IMAGE_NAME}} in envTemplate
 	if v, ok := t.(*envTemplateTagger); ok {
-		tag, err := v.GenerateTag(workingDir, DeprecatedImageName)
+		imageDeprecated := latest.Artifact{}
+		imageDeprecated.ImageName = DeprecatedImageName
+
+		tag, err := v.GenerateTag(imageDeprecated)
```

Comment:
Could inline this (but feel free to ignore)
```suggestion
		tag, err := v.GenerateTag(latest.Artifact{ImageName: DeprecatedImageName})
```

Reply:
changed.